.freeCodeCamp/client/components/heading.tsx

```diff
@@ -5,14 +5,16 @@ interface HeadingProps {
   topic: string;
   title: string;
   lessonNumber?: number;
+  numberOfLessons?: number;
   goToNextLesson?: F<void, void>;
   goToPreviousLesson?: F<void, void>;
 }
 
 export const Heading = ({
   topic,
   title,
-  lessonNumber,
+  lessonNumber = 1,
```

Comment:
I am looking at neatening this up, but we cannot default `lessonNumber`, because `{lessonNumber && ...}` logic is used below